src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -306,7 +306,12 @@ static void startReferenceHandlerThread(ThreadGroup tg) {
         handler.start();
     }
 
+    // Called from JVM when loading an AOT cache
     static {
+        runtimeSetup();
+    }
+
+    private static void runtimeSetup() {
```

Comment:
The comment was added in the wrong spot:
```suggestion
    static {
        runtimeSetup();
    }

    // Called from JVM when loading an AOT cache
    private static void runtimeSetup() {
```

Reply:
Oops, I will fix in an related PR #24979